kukkuu/settings.py

```diff
@@ -143,11 +145,29 @@
 except Exception:
     REVISION = "n/a"
 
+
+def sentry_before_send(event, hint):
+    """
+    Ignore certain errors from being sent to Sentry.
+    NOTE: The `ignore_errors` option is actually deprecated,
+    nevertheless it's included in the init configuration.
+    See https://github.com/getsentry/sentry-python/issues/149
+    """
+
+    IGNORED_ERRORS_CLASSES = (ExpiredSignatureError, AuthenticationExpiredError)
+    if "exc_info" in hint:
+        exc_type, exc_value, tb = hint["exc_info"]
```

Comment:
```suggestion
        exc_type, exc_value, traceback = hint["exc_info"]
```
To make the unused `tb` variable name more transparent & explicit, based on https://docs.python.org/3/library/sys.html#sys.exc_info